src/main/webapp/app/exercises/modeling/manage/example-modeling/example-modeling-submission.component.ts

```diff
@@ -87,9 +87,7 @@ export class ExampleModelingSubmissionComponent implements OnInit {
     private loadAll(): void {
         this.exerciseService.find(this.exerciseId).subscribe((exerciseResponse: HttpResponse<ModelingExercise>) => {
             this.exercise = exerciseResponse.body!;
-            if (this.exercise.exerciseGroup !== null) {
-                this.isExamMode = true;
-            }
+            this.isExamMode = this.exercise.exerciseGroup !== null && this.exercise.exerciseGroup !== undefined ? true : false;
```

Comment:
```suggestion
            this.isExamMode = this.exercise.exerciseGroup != null;
```

Reply:
I learned today that a check with `== null` also includes `== undefined`, so `!= null` should also include `!= undefined` which means we could further simplify the statement, right?

Reply:
ah yes, that is true! But sometimes typescript linter complains about it. I will check that and commit a new version.